setup.py

```diff
@@ -52,7 +52,7 @@
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/<REPO_NAME>",
+    url="https://github.com/soos3d/ape-chainstack",
```

Comment:
Thanks for identifying the missing parameterized url. I think we'll want to change it like so:
```suggestion
    url="https://github.com/ApeWorX/ape-chainstack",
```

Reply:
No problem! I implemented the suggestion 😄